src/buttons/RemovePhotosphere.tsx

```diff
@@ -0,0 +1,69 @@
+import { useState } from "react";
+
+import { VFE } from "../DataStructures";
+import PhotosphereSelector, {
+  PhotosphereSelectorProps,
+} from "../PhotosphereSelector";
+
+interface RemovePhotosphereProps {
+  onClose: () => void;
+  vfe: VFE;
+  onRemovePhotosphere: (PhotosphereID: string) => void; // Function to remove photosphere from navigation map
+}
+
+function RemovePhotosphere({
+  onClose,
+  vfe,
+  onRemovePhotosphere,
+}: RemovePhotosphereProps): JSX.Element {
+  const [selectedPhotosphere, setSelectedPhotosphere] = useState<string>("");
+
+  // Dummy options for the PhotosphereSelector
+  const options: string[] = Object.keys(vfe.photospheres);
+
+  // Create the props for the PhotosphereSelector component
+  const selectorProps: PhotosphereSelectorProps = {
+    options: options,
+    value: selectedPhotosphere,
+    setValue: setSelectedPhotosphere,
+  };
+
+  function HandleRemovePhotosphere() {
+    if (!selectedPhotosphere) {
+      //We need to delete the first photosphere then.
+      if (Object.keys(vfe.photospheres).length > 0) {
+        const photosphereIdRemove = Object.keys(vfe.photospheres)[0];
+        onRemovePhotosphere(photosphereIdRemove);
+      } else {
+        alert("No photospheres to remove.");
+        onClose();
```

Comment:
This behavior is a little bit confusing.

If no photosphere is selected, I would expect that an error shows up instead of deleting the first photosphere in the list.

Alternatively, the photosphere selector default value could be set to the current photosphere.

Reply:
My concern is that the selector is empty by default: 
![image](https://github.com/kingsawpdx/virtualFieldEnvironments/assets/25037249/3c7c153c-92e2-4f22-b273-e4241dc721e0)

I think it should have the name of the first photosphere then.

Reply:
oh yeah you are right, I was remembered the first photosphere would show up by default but I was wrong. We definitely need to change the default to the first photosphere.